### PR TITLE
Add instruction for CLI connection in sui-local-network.md

### PR DESCRIPTION
**Pull request by 666lcz**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: doc/src/build/sui-local-network.md [hunk 1]
```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
 }
 ```
 
+## Connect Command Line Tool to Local Network
```
Comment (randall-Mysten) [inline doc/src/build/sui-local-network.md:152]:
```suggestion
## Connect the Sui Client CLI to your local network
```

File: doc/src/build/sui-local-network.md [hunk 2]
```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
...
+
+```bash
+# If this is your first time creating a local network, create a new environment with alias `local` for local network
```
Comment (randall-Mysten) [inline doc/src/build/sui-local-network.md:155]: If this is the first time they have ever used the client CLI, or the first time after a clean install, they are asked if they want to connect to a Full node, y/N. It uses Devnet by default, and there is no other info about environments. So they'd need to choose a full node and create an address, then set the env alias. I just installed the binaries, started the local network, then used the RPC URL when I configured the client CLI the first time

File: doc/src/build/sui-local-network.md [hunk 3]
```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
...
+sui client new-env --alias local --rpc http://127.0.0.1:9000
+
+# switch to the new local environment
```
Comment (randall-Mysten) [inline doc/src/build/sui-local-network.md:158]:
```suggestion
# set the active environment for the Client CLI to the new local environment
```

File: doc/src/build/sui-local-network.md [hunk 4]
```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
...
+sui client switch --env local
+Active environment switched to [local]
+
+# confirm your local env
+sui client active-env
+local
+
+# show your sui address
```
Comment (randall-Mysten) [inline doc/src/build/sui-local-network.md:166]:
```suggestion
# show the current Sui active address for the Client CLI
```

File: doc/src/build/sui-local-network.md [hunk 5]
```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
...
+sui client active-address
+0xbc33e6e4818f9f2ef77d020b35c24be738213e64d9e58839ee7b4222029610de
+
+# request test tokens from local faucet
```
Comment (randall-Mysten) [inline doc/src/build/sui-local-network.md:170]: This info is already in the section on using the local faucet

**Review by randall-Mysten — APPROVED** [review]

File: doc/src/build/sui-local-network.md (hunk 5 above)
Comment (666lcz) [reply doc/src/build/sui-local-network.md:170]: Yes, I don't know which one to keep, feel free to make a decision and edit directly!

File: doc/src/build/sui-local-network.md (hunk 2 above)
Comment (666lcz) [reply doc/src/build/sui-local-network.md:155]: Yeah feel free to add more clarification about this case

**Pull request merged**